server/interceptor/interceptor.go

```diff
@@ -271,6 +271,29 @@ type rpcInfo struct {
 	tokens int
 }
 
+// chargable is satisfied by request proto messages which contain a GetChargeTo
+// accessor.
+type chargable interface {
+	GetChargeTo() *trillian.ChargeTo
+}
+
+// chargedUsers returns user identifiers for any chargable user quotas.
+func chargedUsers(req interface{}) []string {
+	if req == nil {
+		return nil
+	}
```

Comment:
Drop this arm -- the [type assertion](https://golang.org/ref/spec#Type_assertions) will catch `nil`s